File: MusicPlayer.v2/app/src/main/java/com/example/musicplayerv2/ThemePicker.java
package com.example.musicplayerv2;

import android.content.Context;
import android.content.SharedPreferences;
import android.preference.PreferenceManager;

/**
 * Created by ${Aina} on 13.11.2018.
 */
public class ThemePicker {
    public static int getTheme(Context context) {
        SharedPreferences sharedPref = PreferenceManager.getDefaultSharedPreferences(context);
        String currentTheme = sharedPref.getString("themes", "First");
        int themeId ;
        switch (currentTheme) {
            case "first":
                themeId = R.style.first;
                break;
            case "second":
                themeId = R.style.second;
                break;
            case "third":
                themeId = R.style.third;
                break;
            default:
                themeId = R.style.default1;
        }
        return themeId;
    }
}

File: MusicPlayer.v2Kotlin/app/src/main/java/com/example/musicplayerv2/ThemePicker.kt
package com.example.musicplayerv2

import android.content.Context
import android.content.SharedPreferences
import android.preference.PreferenceManager

/**
 * Created by ${Aina} on 13.11.2018.
 */
object ThemePicker {
    fun getTheme(context: Context): Int {
        val sharedPref = PreferenceManager.getDefaultSharedPreferences(context)
        val currentTheme = sharedPref.getString("themes", "First")
        return when (currentTheme) {
            "first" -> R.style.first
            "second" -> R.style.second
            "third" -> R.style.third
            else -> R.style.default1
        }
    }
}
